server/channels/store/sqlstore/session_store.go

```diff
@@ -123,6 +123,26 @@ func (me SqlSessionStore) GetSessions(c request.CTX, userId string) ([]*model.Se
 	return sessions, nil
 }
 
+func (me SqlSessionStore) GetLRUSessions(c request.CTX, userId string, limit uint64, offset uint64) ([]*model.Session, error) {
```

Comment:
Maybe good to write a comment here saying that it's okay to use LIMIT/OFFSET because they are hardcoded from the app layer.

Reply:
Thanks, done in [426eb05](https://github.com/mattermost/mattermost/pull/25900/commits/426eb0514266b5ba861179635d8d34141c565dfe)